src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
+=====================
+
+..
+  gitlab.com > create account
+
+Project Members
+===================
+
+In the local development you must update the git files of each project to the new Gitlab repository.
+Therefore, you only have to write this in the terminal inside the project directory:
+
+.. code:: bash
+
+  $ git remote set-url https://gitlab.com/ska-telescope/*project*.git
+
+
+Project Owners
+==================
+
+Create Gitlab Project
+---------------------------
+
+  
+..
+  gitlab.com > new project >  import project > github > login > select the project you want to import
+
+
+Mirror Repository
+-----------------
+
+..
+  For an existing project, you can set up push mirroring as follows:
+
+  Navigate to your project’s Settings > Repository and expand the Mirroring repositories section.
+  Enter a repository URL.
+  Select Push from the Mirror direction dropdown.
+  Select an authentication method from the Authentication method dropdown, if necessary.
+  Check the Only mirror protected branches box, if necessary.
+  Click the Mirror repository button to save the configuration.
+.. 
+  Setting up a push mirror from GitLab to GitHub
+  To set up a mirror from GitLab to GitHub, you need to follow these steps:
+
+  Create a GitHub personal access token with the public_repo box checked.
+  Fill in the Git repository URL field using this format: https://<your_github_username>@github.com/<your_github_group>/<your_github_project>.git.
+  Fill in Password field with your GitHub personal access token.
+  Click the Mirror repository button.
+  The mirrored repository will be listed. For example, https://*****:*****@github.com/<your_github_group>/<your_github_project>.git.
+
+  The repository will push soon. To force a push, click the appropriate button.
+
+Read-Only GitHub Project
```

Comment:
I'd add something like "Once switched over, you can make the GitHub project Read-Only. We recommend doing this, so that you can ensure that active development takes place on GitLab only."